src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/CreateFileFromTemplateService.cs

```diff
@@ -52,7 +52,7 @@ public CreateFileFromTemplateService(IUnconfiguredProjectVsServices projectVsSer
             {
                 HierarchyId parentId = _projectVsServices.VsProject.GetHierarchyId(parentDocumentMoniker);
                 var result = new VSADDRESULT[1];
-                var files = new string[] { templateFilePath };
+                string[] files = new string[] { templateFilePath };
```

Comment:
feels like type is apparent here

Reply:
This looks like: https://github.com/dotnet/roslyn/issues/23907, but a different repro, will file.

Reply:
Not going to fix this until the Roslyn bug is fixed.

Reply:
➡️ Not a bug. This false into the built-in type category.

https://github.com/dotnet/project-system/blob/5f7f5321737cc15290ff7b69e0377573c4a1a6a8/.editorconfig#L123

Reply:
👍

Reply:
I filed https://github.com/dotnet/roslyn/issues/23714 to revisit the design. It's odd that you effectively cannot turn on only "when variable type is apparent" and have it apply to built-in types.